include/swift/Option/Options.td

```diff
@@ -310,6 +313,31 @@ def AssertConfig : Separate<["-"], "assert-config">,
   HelpText<"Specify the assert_configuration replacement. "
            "Possible values are Debug, Release, Unchecked, DisableReplacement.">;
 
+// Code formatting options
+
+def code_formatting_Group : OptionGroup<"<code formatting options>">;
+
+def use_tabs : Flag<["-"], "usetabs">, Group<code_formatting_Group>,
+  Flags<[SwiftFormatOption]>,
+  HelpText<"Use tabs for indentation.">;
+
+def format_inplace : Flag<["-"], "inplace">, Group<code_formatting_Group>,
```

Comment:
`format_inplace` makes be think that the option should be spelt `format-inlace`.